api/src/pcapi/core/mails/transactional/pro/offer_validation_to_pro.py

```diff
@@ -67,21 +67,27 @@ def retrieve_data_for_validated_offer_rejection_email(
     )
 
 
-def send_offer_validation_status_update_email(
+def get_email_data_from_offer(
     offer: Offer | educational_models.CollectiveOffer | educational_models.CollectiveOfferTemplate,
     old_status: OfferValidationStatus,
     validation_status: OfferValidationStatus,
-    recipient_emails: list[str],
-) -> None:
+) -> models.TransactionalEmailData:
     if validation_status is OfferValidationStatus.APPROVED:
         offer_data = retrieve_data_for_offer_approval_email(offer)
-        mails.send(recipients=recipient_emails, data=offer_data)
+        return offer_data
+
+    # REJECTED
```

Comment:
Par sécurité j'aurais gardé `if validation_status is OfferValidationStatus.REJECTED:`.
Même si ce n'est appelé en l'état actuel que dans les deux cas APPROVED et REJECTED.